benchmarks/benchmark/tools/latency-profile/manifest-templates/hf-secret.yaml.tpl

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: hf-token
+  namespace: ${namespace}
+data:
+  HF_TOKEN: ${hugging_face_token_b64}
```

Comment:
This looks like automation for creating the k8s hardcoded secret pathway (an option that enables customers to use their own k8s secret instead of using secret manager) that uses k8s_hf_secret: https://github.com/GoogleCloudPlatform/ai-on-gke/blob/1f4c96861ad166fd8c2c7c1847e45c8b187e362c/benchmarks/benchmark/tools/locust-load-inference/variables.tf#L201

Long term I think we should consider using the same implementation pathway as locust does for this (we can keep the creation automation as well), and also make sure we support both pathways (secret manager and k8s secret)